test/src/tests/search.test.js

```diff
@@ -0,0 +1,51 @@
+import {
+  CLIENT_URL,
+  uniqueId,
+  registerNewUser,
+  registerDandiset,
+} from '../util';
+import * as homePage from '../pages/homePage';
+import * as searchPage from '../pages/searchPage';
+
+
+describe('dandiset search page', () => {
+  const name = `test${uniqueId()}`;
+  const description = `Description ${uniqueId()}`;
+
+  beforeAll(async () => {
+    await page.goto(CLIENT_URL);
+    await registerNewUser();
+
+    // create a new dandiset that will appear in search results
+    await registerDandiset(name, description);
```

Comment:
I worry that this shared setup is starting to tread close to the inter-test shared state that [the `jest/no-hooks` ESLint rule](https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/no-hooks.md) is trying to prevent. In this case, I think it's acceptable, since none of the test cases mutate the server state, but I wonder what you think in general.